src/Microsoft.Dnx.Host.Clr/project.json

```diff
@@ -1,8 +1,14 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": { "define": [ "TRACE" ], "allowUnsafe": true, "warningsAsErrors": true, "keyFile": "KeyFile.snk" },
-    "dependencies": {
-    },
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "define": [ "TRACE" ],
+    "allowUnsafe": true,
+    "warningsAsErrors": true,
+    "keyFile": "KeyFile.snk"
+  },
+  "dependencies": {
+    "Microsoft.Dnx.Runtime.Json.Sources": { "version": "1.0.0-*", "type": "build" }
```

Comment:
I know why we need it - recent changes to set .NET Framework version. I wonder if you can fake strong name `Microsoft.Dnx.Runtime.Json.Sources` to make it work. I believe the plan is to fake strong name all assemblies anyways.